### PR TITLE
Mn/setproject

### PR DESCRIPTION
**Pull request by matydn05**
Setting folders up for the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,11 @@
     "express": "^4.16.4",
     "graphql": "^14.1.1",
     "http": "0.0.0",
-    "lodash": "^4.17.11"
+    "lodash": "^4.17.11",
+    "pg": "^7.10.0",
+    "sequelize": "^5.8.5",
+    "sequelize-cli": "^5.4.0",
+    "sqlite3": "^4.0.6"
```
Comment (abelosorio) [inline package.json:29]: This is not used anymore, right? In that case, please remove it from the package.json.

File: src/config/config.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "root",
+    "password": null,
+    "database": "database_development",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
```
Comment (abelosorio) [inline src/config/config.json:7]: dialect should be `postgres`.

File: package.json (hunk 1 above)
Comment (matydn05) [reply package.json:29]: Right, this was removed from the package.json in the more update mn/setdatabase branch.

File: src/config/config.json (hunk 2 above)
Comment (matydn05) [reply src/config/config.json:7]: Changed to postgre dialet while I was setting up the DB, you should see it fixed in mn/setdatabase.

**Comment by matydn05** [comment]
Ready to merge ?

**Pull request merged**